android/app/src/main/java/org/fossasia/openevent/viewmodels/EditProfileActivityViewModel.java

```diff
@@ -0,0 +1,125 @@
+package org.fossasia.openevent.viewmodels;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+
+import org.fossasia.openevent.api.APIClient;
+import org.fossasia.openevent.data.auth.UploadImage;
+import org.fossasia.openevent.data.auth.User;
+import org.fossasia.openevent.dbutils.RealmDataRepository;
+import org.fossasia.openevent.utils.JWTUtils;
+import org.fossasia.openevent.utils.Utils;
+import org.json.JSONException;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+import timber.log.Timber;
+
+public class EditProfileActivityViewModel extends ViewModel {
+
+    public static final int IMAGE_UPLOAD_SUCCESSFUL =1;
+    public static final int IMAGE_UPLOAD_FAILED =2;
+    public static final int IMAGE_UPLOAD_COMPLETE =3;
+    public static final int USER_UPDATE_SUCCESSFUL=1;
+    public static final int USER_UPDATE_FAILED=2;
+    public static final int USER_UPDATE_COMPLETE=3;
+
+    private RealmDataRepository realmRepo;
+    private User user;
+    private LiveData<User> userLiveData;
+    private String uploadedImageUrl;
+    private MutableLiveData<Integer> imageUploadResponse;
+    private MutableLiveData<Integer> userUpdateResponse;
+    private CompositeDisposable compositeDisposable;
+
+    public EditProfileActivityViewModel(){
+        realmRepo = RealmDataRepository.getDefaultInstance();
+        compositeDisposable = new CompositeDisposable();
+    }
+
+    public LiveData<User> getUser(){
+        if(userLiveData == null) {
+            user = realmRepo.getUser();
+            userLiveData = RealmDataRepository.asLiveDataForObject(user);
+        }
+        return userLiveData;
+    }
+
+    public LiveData<Integer> uploadImage(String encodedImage){
+        if(imageUploadResponse == null){
+            imageUploadResponse = new MutableLiveData<>();
+        }
+        Disposable uploadDisposable;
+        uploadDisposable = APIClient.getOpenEventAPI().uploadImage(new UploadImage(encodedImage))
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(imageResponse -> {
+                    uploadedImageUrl = imageResponse.getUrl();
+                    imageUploadResponse.setValue(IMAGE_UPLOAD_SUCCESSFUL);
+                    Timber.d("Image uploaded successfully." + " Url: " + uploadedImageUrl);
+                },throwable -> {
+                    imageUploadResponse.setValue(IMAGE_UPLOAD_FAILED);
+                    Timber.d("Error uploading image: " + throwable.getMessage());
+                },() -> {
+                    imageUploadResponse.setValue(IMAGE_UPLOAD_COMPLETE);
+                });
+
+        compositeDisposable.add(uploadDisposable);
+
+        return imageUploadResponse;
+
```

Comment:
Remove this line and line 71 also. I believe line 69 is okay, but the rest aren't needed.